operator_ui/src/components/Bridges/Form.tsx

```diff
@@ -1,42 +1,63 @@
-import React from 'react'
-import PropTypes from 'prop-types'
-import { Prompt } from 'react-router-dom'
-import * as formik from 'formik'
-import { withStyles } from '@material-ui/core/styles'
-import { TextField, Grid } from '@material-ui/core'
+import { Grid, TextField } from '@material-ui/core'
+import {
+  createStyles,
+  Theme,
+  withStyles,
+  WithStyles
+} from '@material-ui/core/styles'
 import Button from 'components/Button'
-import { set, get } from 'utils/storage'
+import formik, { FormikProps } from 'formik'
 import normalizeUrl from 'normalize-url'
+import React from 'react'
+import { Prompt } from 'react-router-dom'
+import { get, set } from 'utils/storage'
```

Comment:
I'd suggest converting this util function to typescript too as its not too big and closely related to this component.

Reply:
Yeah, ill be converting this in another commit. Trying to keep commits atomic as possible since this PR will grow quite big